lib/models/item_menu_action.dart

```diff
@@ -133,6 +139,8 @@ extension ItemMenuActionExtension on ItemMenuAction {
             : FluentIcons.clipboard_text_ltr_24_regular;
       case ItemMenuAction.share:
         return FluentIcons.share_24_regular;
+      case ItemMenuAction.toggleVisited:
+        return FluentIcons.eye_24_regular;
```

Comment:
Good choice, but let's use `FluentIcons.eye_off_24_regular` in case the toggle would mark the item as unvisited!